public/script.js

```diff
@@ -6,6 +6,38 @@ $( document ).ready(function() {
   canvas.height = window.innerHeight;
   drawStuff();
 
+  $("#download").click(function() {
+    var outputCanvas = document.createElement("canvas");
+    outputCanvas.width = canvas.width;
+    outputCanvas.height = canvas.height;
+
+    var outputContext = outputCanvas.getContext("2d");
+    outputContext.font = "bold 24pt Times";
+
+    var gradient = outputContext.createLinearGradient(0, 0, canvas.width, 0);
+    gradient.addColorStop(0, "#ECF9FF");
```

Comment:
The gradient being on the canvas would hide the text :(
Is OK, I'll get the gradient info from the variables. Yep we can do angles too, I'll have to dust off my sixth form trigonometry! Will do so in next couple of days.
On 23 Jul 2015 10:46 pm, Merrin Macleod <notifications@github.com> wrote:In public/script.js:
> @@ -6,6 +6,38 @@ $( document ).ready(function() {
>    canvas.height = window.innerHeight;
>    drawStuff();
>  
> +  $("#download").click(function() {
> +    var outputCanvas = document.createElement("canvas");
> +    outputCanvas.width = canvas.width;
> +    outputCanvas.height = canvas.height;
> +
> +    var outputContext = outputCanvas.getContext("2d");
> +    outputContext.font = "bold 24pt Times";
> +
> +    var gradient = outputContext.createLinearGradient(0, 0, canvas.width, 0);
> +    gradient.addColorStop(0, "#ECF9FF");

Hmm, so the initial goal was to change the background gradient on a daily basis. I never ended up automating that but the colours are now defined in index.js as background_colours - it'd be good to avoid having to define those colours twice. 

Additionally, I've been changing the angle of the background gradient as well as the colours - this won't be able to do that, will it? 

Perhaps it's possible to have the gradient background on the canvas all along? 

—Reply to this email directly or view it on GitHub.